src/librustc_typeck/check/coercion.rs

```diff
@@ -1163,18 +1163,20 @@ impl<'tcx, 'exprs, E: AsCoercionSite> CoerceMany<'tcx, 'exprs, E> {
                 fcx.try_coerce(expression, expression_ty, self.expected_ty, AllowTwoPhase::No)
             } else {
                 match self.expressions {
-                    Expressions::Dynamic(ref exprs) =>
-                        fcx.try_find_coercion_lub(cause,
-                                                  exprs,
-                                                  self.merged_ty(),
-                                                  expression,
-                                                  expression_ty),
-                    Expressions::UpFront(ref coercion_sites) =>
-                        fcx.try_find_coercion_lub(cause,
-                                                  &coercion_sites[0..self.pushed],
-                                                  self.merged_ty(),
-                                                  expression,
-                                                  expression_ty),
+                    Expressions::Dynamic(ref exprs) => fcx.try_find_coercion_lub(
```

Comment:
Btw, in general, I would leave rustfmt-ing alone since we will rustfmt the whole repo eventually anyways...

Reply:
Usually I do formatting close to the code I'm touching. Often during writing a PR I back out of a functional change but leave the formatting changes I made seeing them as a no-op that leaves the codebase a bit cleaner than before :)

I can certainly stop doing that.

Reply:
Not a big deal anyways =) I sometimes cannot resist the urge to format some myself so I should take my own advice. ;)